### DISCUSSION TITLE
Galaxy Badge not showing up

### DISCUSSION THREAD

**Discussion by pedrolemos4** [community/community#127574] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have 2 marked correct answers to 2 different discussions [here](https://github.com/orgs/community/discussions/127417) and [here](https://github.com/orgs/community/discussions/126737), why does the Galaxy Badge does not show up on my profile?

**Comment by facundoprocelli** [comment]
You must do it on 0 answers commetns

**Comment by davevad93** [comment, marked as answer]
Hi @pedrolemos4 ,

Unfortunately the `Galaxy Brain` badge is no longer obtainable via Community Discussions, for more info see [here](https://github.com/orgs/community/discussions/106536), however it can still be earned through discussions in other `public` repos.

**Reply by pedrolemos4** [reply]
-_-
What do you mean with public repos?

**Reply by davevad93** [reply]
There are `public` repositories on GitHub that have open discussions, the truth is that I couldn't specify which ones but exploring certain repos I've noticed some of them have a discussion section. Everyone can enable discussions in their own repository, if you wanna know how to do it follow these steps:

1. Go to `Settings`.
2. In the `General` tab scroll down until you see the `Discussions` checkbox.
3. Activate it.

![0](https://github.com/community/community/assets/94308234/a5f54fd5-923d-4e0e-8df8-8779bd252b09)

**Reply by pedrolemos4** [reply]
Alright, thanks for the help :)

**Reply by davevad93** [reply]
You're welcome man, have a nice day!
